media/css/firefox/new/fx-lifestyle/browse-up-to-you.less

```diff
@@ -5,6 +5,7 @@
 @import '../../../sandstone/lib.less';
 
 #intro {
+    background-color: #868b8f;
```

Comment:
Added some background colors that match the pallet of each image. I noticed while working out in the garden that the pages are unreadable until the images load, which will likely hurt slower connections :/